aptos-move/aptos-vm-types/src/write_change_set.rs

```diff
@@ -0,0 +1,118 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::change_set::ChangeSet;
+use aptos_aggregator::delta_change_set::DeltaChangeSet;
+use aptos_state_view::StateView;
+use aptos_types::{
+    state_store::state_key::StateKey,
+    write_set::{WriteOp, WriteSet, WriteSetMut},
+};
+use move_core_types::vm_status::{StatusCode, VMStatus};
+use std::{
+    collections::btree_map::{
+        Entry::{Occupied, Vacant},
+        IntoIter,
+    },
+    ops::{Deref, DerefMut},
+};
+
+/// All writes produced by the VM.
+#[derive(Debug, Clone)]
+pub struct WriteChangeSet(ChangeSet<WriteOp>);
```

Comment:
In the next PR let's make it generic:  `WriteChangeSet<T: Store>(ChangeSet<Op<T>>)`.